BlinkStickDotNet/Colors/HSLColor.cs

```diff
@@ -0,0 +1,174 @@
+using System;
+
+namespace BlinkStickDotNet.Colors
+{
+    /// <summary>
+    /// Defines an HSL color.
+    /// </summary>
+    /// <remarks>Check http://en.wikipedia.org/wiki/HSL_color_space for more info.</remarks>
+    public struct HSLColor
```

Comment:
Ok -- I see that this type was moved, not added. Comment applies to upstream code, in that case.